qiskit/result/counts.py

```diff
@@ -183,3 +183,11 @@ def int_outcomes(self):
     def _remove_space_underscore(bitstring):
         """Removes all spaces and underscores from bitstring"""
         return int(bitstring.replace(" ", "").replace("_", ""), 2)
+
+    def size(self):
+        """Return the number of clbits"""
+        return len(next(iter(self)))
```

Comment:
Does this work if counts is stored with integer keys? or has white space in the bitstring keys?
I dont think `size` is a good name here either, since I would expect it to correspond to length of hte dict, not its keys. Adding `num_clbits` as a property would be better, since this is what is used for circuits / instructions etc

Reply:
I removed this method, as it is not used in the mitigation class code.